Sources/MapboxNavigation/EndOfRouteFeedbackCollecting.swift

```diff
@@ -0,0 +1,142 @@
+import UIKit
+import MapboxDirections
+import MapboxCoreNavigation
+import MapboxMobileEvents
+
+
+public protocol EndOfRouteFeedbackCollecting: UIViewController {
+    typealias EndOfRouteDismissalHandler = (EndOfRouteFeedback?) -> ()
+    
+//    private var endOfRouteViewController: EndOfRouteViewController // expose via protocol?
```

Comment:
`EndOfRouteViewController` is not available publicly(by the way, why?). So here we could either expose this class, or introduce protocol stub to allow users to provide their own custom `EndOfRouteViewController`'s versions.